rest-server/admin/src/main/scala/com/actionml/admin/MongoAdministrator.scala

```diff
@@ -177,3 +185,11 @@ case class EngineMetadata(
   engineId: String,
   engineFactory: String,
   params: String)
+
+// not sure how to bring this in scope since it is defined during the compile step
+// of build.sbt
+/*
+object BuildInfo {
+  val version = "${version.value}"
+}
+*/
```

Comment:
this is the same as the definition in build.sbt

@andreyshikov how do we bring this global object in scope? Notice I changed it to be in com.actionml.admin